src/operation/operation_model_interface.jl

```diff
@@ -104,13 +104,24 @@ end
 
 function solve_impl!(model::OperationModel)
     container = get_optimization_container(model)
+    model_name = get_name(model)
+    ts = get_current_timestamp(model)
+    output_dir = get_output_dir(model)
+
+    if get_export_optimization_model(get_settings(model))
+        model_output_dir = joinpath(output_dir, "optimization_model_exports")
+        if !ispath(model_output_dir)
```

Comment:
In case you care, this is slightly more convenient.
```suggestion
        mkpath(model_output_dir)
```